experimental/dds/tree2/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -424,6 +424,45 @@ describe("SharedTree benchmarks", () => {
 			});
 		}
 	});
+
+	// Note that the times reported by this benchmark represent the computation of several peers.
+	// In practice, this computation is distributed across peers, so the actual time reported should be
```

Comment:
If that's the case, could we actually take the `duration` and divide it by `nbPeers` before we pass it to `recordBatch`?